amberly-awang617/week-4/mongoose-book-app/models/index.js

```diff
@@ -0,0 +1,5 @@
+const mongoose = require("mongoose");
+mongoose.connect("mongodb://localhost/book-app", {useNewUrlParser: true});
+
+module.exports.Book = require('./book.js');
+module.exports.Author = require('./author.js');
```

Comment:
This method works fine with exporting your modules, but adjusting it into an object would allow you it to be easier to follow. 
```
module.exports = {
Book: require('./book.js'),
Author: require('./author.js')
}
```
Just a little easier to read and you know what you are accessing in your other files.